src/main/java/eu/darkbot/api/items/ItemUseResult.java

```diff
@@ -0,0 +1,65 @@
+package eu.darkbot.api.items;
+
+import eu.darkbot.api.managers.HeroItemsAPI;
+
+import java.util.function.Consumer;
+
+/**
+ * {@link HeroItemsAPI#useItem(SelectableItem, ItemUseFlag...)} results
```

Comment:
What a class does is not defined by where it is used,  you can still link or @see it but it is not the definition

`The result after attempting to use an in-game item, usually via {@link ...}`